charts/consul/test/unit/mesh-gateway-deployment.bats

```diff
@@ -1783,4 +1783,21 @@ key2: value2' \
   local actual=$(echo "$cmd" |
     yq 'any(contains("-log-level=warn"))' | tee /dev/stderr)
   [ "${actual}" = "true" ]
+}
+
+#--------------------------------------------------------------------
+# security context
```

Comment:
We don't need to add tests for things that aren't conditional since it's basically just a tautology at that point. We just add them for things that are conditional